File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/library/request/read/BroadcastReadPermissionsRequest.java
package com.lasthopesoftware.bluewater.client.browsing.library.request.read;

import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId;

public interface BroadcastReadPermissionsRequest {
	void sendReadPermissionsRequestedBroadcast(LibraryId libraryId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/library/request/read/BroadcastReadPermissionsRequest.kt
package com.lasthopesoftware.bluewater.client.browsing.library.request.read

import com.lasthopesoftware.bluewater.client.browsing.library.repository.LibraryId

fun interface BroadcastReadPermissionsRequest {
    fun sendReadPermissionsRequestedBroadcast(libraryId: LibraryId)
}
